docker/backend/Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM node:12 as base
 WORKDIR /usr/src/app
 
 ARG GRAPHQL_HOST
+ARG GRAPHQL_DOMAIN=onrender.com
```

Comment:
> add this env variable to our render.com account ?

What exactly do you mean by that?
If you mean actually setting it on build then no, since it defaults to "onrender.com", which it also was before this change.